src/Components/v2/__tests__/CountdownTimer.test.tsx

```diff
@@ -95,7 +95,7 @@ describe("CountdownTimer", () => {
 
       const text = timer.text()
       expect(text).toMatchInlineSnapshot(
-        `"time remaining01d 00s leftRespond by Aug 04, 2:50 PM BSTExpired offers end the negotiation process permanently."`
```

Comment:
Reaction is still using Moment's API to mock the timezone, Palette is now using Luxon. Once we merge this PR we will update these tests.